.github/workflows/testing.yml

```diff
@@ -0,0 +1,44 @@
+name: Full Tests
+on:
+  schedule:
+    - cron: "0 16 * * *"  # Daily at noon EST
+  pull_request:
```

Comment:
Ohh right, two things here that would be really convenient

1) adjust trigger condition here to `workflow_call` following the pattern of https://github.com/NeurodataWithoutBorders/nwbinspector/blob/dev/.github/workflows/testing.yml#L2; this will allow the true trigger conditions to be like https://github.com/NeurodataWithoutBorders/nwbinspector/blob/dev/.github/workflows/deploy-tests.yml#L1-L35 which is faster by skipping tests when nothing important has changed (i.e., something minor in the README) and by allowing (2)

(2) in a separate cron-based trigger, such as https://github.com/NeurodataWithoutBorders/nwbinspector/blob/dev/.github/workflows/dailies.yml, note that last part about notification. If you follow that pattern but replace the secret "MAIL_LIST" with just your and mine catalystneuro emails, comma-separated (since they are publicly exposed as contacts in the pyproject, why not...); that way if the dailies start failing all of a sudden we'll both get notified about it

Reply:
Ah, super cool! Thanks for pointing this out.